src/todos/todos.service.ts

```diff
@@ -8,8 +8,11 @@ import { TodoRepository } from './todos.repository';
 export class TodosService {
   constructor(private todosRepository: TodoRepository) {}
 
-  async findAll(): Promise<Todo[]> {
-    return await this.todosRepository.findAll();
+  async findAll(
+    sortBy: string = 'createdAt',
+    sortOrder: string = 'asc',
+  ): Promise<Todo[]> {
+    return await this.todosRepository.findAllSorted(sortBy, sortOrder);
```

Comment:
Food For Thought (FFT) - I think findAllSorted is an extra layer of abstraction here. We could have just defined it in the service layer and called findAllSorted in the controller.